recipes-kernel/linux/topic-xilinx-kernel-patches.inc.orig

```diff
@@ -0,0 +1,82 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-xlnx:"
```

Comment:
I think this is checked in by accident. (.orig)

Reply:
Yeah, good catch